src/main/python/measure_judged.py

```diff
@@ -0,0 +1,69 @@
+"""Script to measure the percentage of judged documents at various
+cutoffs."""
+import argparse
+import collections
+
+from typing import Dict
+from typing import List
+from typing import Set
+
+
+def load_qrels(path: str) -> Dict[str, Set[str]]:
+    """Loads qrels into a dict of key: query_id, value: set of relevant doc
```

Comment:
I don't think these line breaks are necessary? i.e., these lines aren't too long?

Reply:
I'm following the 80-char limit from PEP8. BTW, should we add linting to the repo?